src/sentry/static/sentry/app/views/performance/transactionDetails/quickTrace.tsx

```diff
@@ -335,14 +331,19 @@ function EventNodeSelector({
   const errors: TraceError[] = [];
   events.forEach(e => {
     e?.errors?.forEach(error => {
-      errors.push({
-        ...error,
-        transaction: e.transaction,
-      });
+      if (!currentEvent || currentEvent.id !== error.event_id) {
+        errors.push({
+          ...error,
+          transaction: e.transaction,
+        });
+      }
     });
   });
+  // Filter out the current event so its not in the dropdown
+  events = currentEvent ? events.filter(e => e.event_id !== currentEvent.id) : events;
+
   let type: keyof Theme['tag'] = nodeKey === 'current' ? 'black' : 'white';
-  if (errors.length > 0) {
+  if (errors.length > 0 || (currentEvent && currentEvent.type !== 'transaction')) {
```

Comment:
```suggestion
  if (errors.length > 0 || (currentEvent?.type !== 'transaction')) {
```

Reply:
Realized I had to include this, cause currentEvent is only defined for the current Node, without the check meant that other nodes without errors would think they had an error.